init.sh

```diff
@@ -168,10 +163,18 @@ Server = http://repo.archlinux.fr/$arch'\'' >> /etc/pacman.conf'
         the_silver_searcher \
         tmux \
         wget \
-        yaourt \
         zsh \
         zsh-completions
-    yaourt -Sy --noconfirm --needed \
+
+    # Install Yay if not installed
+    if [ ! -x "$(command -v yay)"]; then
+        printf '\e[1mInstalling Yay\e[0m\n'
+        git clone https://aur.archlinux.org/yay.git && cd yay
```

Comment:
I would prefer to do this work in a `/tmp` folder instead of manually having to remove it afterwards. What do you think of the following?

```bash
git clone https://aur.archlinux.org/yay.git /tmp/yay
(cd /tmp/yay && makepkg -si)
```

Reply:
Yeah ᕙ(⇀‸↼‶)ᕗ, you are right! I will change this line of code as soon as possible!

Reply:
By the way, the repository is wrong :( . The right repo from which to clone Yay is _https://github.com/Jguer/yay.git_

Reply:
I think the current one is correct. At least that's what they list in their README: https://github.com/Jguer/yay#installation

Reply:
I have pushed the code change! (｡◕‿◕｡)